src/osd/OSD.cc

```diff
@@ -7906,8 +7906,8 @@ void OSD::trim_maps(epoch_t oldest, int nreceived, bool skip_maps)
       int tr = store->queue_transaction(service.meta_ch, std::move(t), nullptr);
       ceph_assert(tr == 0);
       num = 0;
-      if (!skip_maps) {
-	// skip_maps leaves us with a range of old maps if we fail to remove all
+      if (!trim_all_maps) {
+	// trim_all_maps leaves us with a range of old maps if we fail to remove all
 	// of them before moving superblock.oldest_map forward to the first map
 	// in the incoming MOSDMap msg. so we should continue removing them in
 	// this case, even we could do huge series of delete transactions all at
```

Comment:
(I know it's not part of the change set - but can you rewrite/rephrase this comment?)